helpdesk_mgmt_project/models/helpdesk_ticket.py

```diff
@@ -5,7 +5,9 @@ class HelpdeskTicket(models.Model):
 
     _inherit = "helpdesk.ticket"
 
-    project_id = fields.Many2one(string="Project", comodel_name="project.project")
+    project_id = fields.Many2one(
+        "project.project", string="Project", tracking=True, check_company=True
```

Comment:
please keep comodel_name, it ease the readability even if technically it's correct